src/Bicep.MSBuild.E2eTests/src/example.ts

```diff
@@ -0,0 +1,72 @@
+
+import * as spawn from 'cross-spawn';
+import * as path from "path";
+import * as fs from "fs";
+import { SpawnSyncReturns } from 'node:child_process';
```

Comment:
[nit] Inconsistent quotes. Formatting the document in VSCode will fix these automatically if you have the prettier extension installed.

Reply:
Fixed by auto formatting.